CHANGELOG.md

```diff
@@ -273,11 +275,58 @@ been added to each release, please refer to the [blog](https://blog.gitea.io).
   * Add docs command (#13429)
   * Update external-renderers.en-us.md (#13165)
 * MISC
+  * Add size to Save function (#15264) (#15270)
```

Comment:
I think this is an Enhancement at least, if not bug fix - the reality is that it produces a significant performance improvement on minio storage.